docs/source/daml/daml-studio.rst

```diff
@@ -319,9 +319,9 @@ choice 'Consume' of which he is not a controller
       TX #0 1970-01-01T00:00:00Z (unknown source)
       #0:0
       │   disclosed to (since): 'Alice' (#0), 'Bob' (#0)
-      └─> create ExerciseAuthFailure:Example
-          with
-            owner = 'Alice'; friend = 'Bob'
+      └─> 'Alice' creates ExerciseAuthFailure:Example
```

Comment:
looking at this I noticed a difference between the haskell and scala pretty printers: the former quotes party values while the latter (sometimes) doesn't. I think it's easier to read with the quotes. In any case, this would be another PR.

Reply:
I'm actually rewriting that bit rn, so its a trivial change I can add if we want it?

Reply:
or do we think it needs more discussion

Reply:
oh actually, theres a lot more cases than i thought, agree separate PR lol

Reply:
> or do we think it needs more discussion

not really, it's more about keeping the changes easier to follow